packages/aragon-wrapper/src/index.js

```diff
@@ -1540,7 +1558,7 @@ export default class Aragon {
    * @param  {Array<Object>} path
    * @return {Promise<Array<Object>>} The given `path`, with decorated with descriptions at each step
    */
-  describeTransactionPath (path) {
+  async describeTransactionPath (path) {
```

Comment:
I think this can be left non-async, since it already returns a promise

Reply:
true, but sometimes it's useful to tell if the the functions is actually returning a promise or not from its definition adding the `async` keyword, otherwise you will need to read the function to tell if that's the case... but I can remove it if we already decided to do the other way around :)

Reply:
No real preference, and the code styling on aragon.js is a bit awkward compared to the other repos we have (to be fixed soon). Generally I think we avoid `async` unless there's an explicit `await` :shrug:.